alicloud/resource_alicloud_cen_bandwidth_package.go

```diff
@@ -127,31 +127,31 @@ func resourceAlicloudCenBandwidthPackage() *schema.Resource {
 func resourceAlicloudCenBandwidthPackageCreate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*connectivity.AliyunClient)
 	cenService := CenService{client}
-	var cenbwp *cbn.CreateCenBandwidthPackageResponse
+	var response *cbn.CreateCenBandwidthPackageResponse
+	request := buildAliCloudCenBandwidthPackageArgs(d, meta)
 	err := resource.Retry(3*time.Minute, func() *resource.RetryError {
-		args := buildAliCloudCenBandwidthPackageArgs(d, meta)
 		raw, err := client.WithCenClient(func(cbnClient *cbn.Client) (interface{}, error) {
-			return cbnClient.CreateCenBandwidthPackage(args)
+			return cbnClient.CreateCenBandwidthPackage(request)
 		})
 		if err != nil {
 			if IsExceptedError(err, OperationBlocking) {
-				return resource.RetryableError(fmt.Errorf("Create CEN bandwidth package timeout and got an error: %#v.", err))
+				return resource.RetryableError(err)
 			}
 			return resource.NonRetryableError(err)
 		}
-		resp, _ := raw.(*cbn.CreateCenBandwidthPackageResponse)
-		cenbwp = resp
+		response = raw.(*cbn.CreateCenBandwidthPackageResponse)
+		addDebug(request.GetActionName(), raw)
 		return nil
 	})
 	if err != nil {
-		return fmt.Errorf("Create CEN bandwidth package got an error :%#v", err)
+		return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
 	}
 
-	d.SetId(cenbwp.CenBandwidthPackageId)
-
-	err = cenService.WaitForCenBandwidthPackage(d.Id(), Idle, DefaultCenTimeout)
+	d.SetId(response.CenBandwidthPackageId)
+	// set bandwidth "-1" here to use WaitForCenBandwidthPackage, actually determined by status.
+	err = cenService.WaitForCenBandwidthPackage(d.Id(), Idle, -1, DefaultCenTimeout)
```

Comment:
I think the line 152 should be `err = cenService.WaitForCenBandwidthPackage(d.Id(), Idle, request.Bandwidth, DefaultCenTimeout)`